reconcile/sql_query.py

```diff
@@ -366,7 +353,12 @@ def encrypted_closing_message(recipient):
     ]
 
 
-def process_template(query, image_repository, use_pull_secret=False):
+def process_template(
+    query: dict[str, Any],
+    image_repository: str,
+    use_pull_secret: bool,
+    config_map_names: list[str],
+) -> str:
```

Comment:
After the change, the below doc is no longer accurate.

Reply:
👍